discord/commands/core.py

```diff
@@ -155,16 +155,14 @@ def unwrap_function(function: Callable[..., Any]) -> Callable[..., Any]:
 
 def _validate_names(obj):
     validate_chat_input_name(obj.name)
-    if obj.name_localizations:
-        for locale, string in obj.name_localizations.items():
-            validate_chat_input_name(string, locale=locale)
+    for locale, string in obj.name_localizations.items():
+        validate_chat_input_name(string, locale=locale)
```

Comment:
This needs to be reverted because it defaults to `MISSING` now
```suggestion
    if obj.name_localizations:
        for locale, string in obj.name_localizations.items():
            validate_chat_input_name(string, locale=locale)
```